osu.Game/Rulesets/Difficulty/DifficultyAttributes.cs

```diff
@@ -25,6 +25,8 @@ public class DifficultyAttributes
         protected const int ATTRIB_ID_SCORE_MULTIPLIER = 15;
         protected const int ATTRIB_ID_FLASHLIGHT = 17;
         protected const int ATTRIB_ID_SLIDER_FACTOR = 19;
+        protected const int ATTRIB_ID_AIM_DIFFICULT_STRAIN_COUNT = 21;
```

Comment:
is this really intended to be duplicate of `ATTRIB_ID_SPEED_NOTE_COUNT `?

Reply:
This is a good (and *very* scary) catch. I can see how this could have happened, the speed note count thing was added in https://github.com/ppy/osu/pull/15035 which precedes the open date of this PR.

That said I'm not sure I would trust *any* sheet results at this point given this revelation because I'm not sure what this means for correct calculation. Probably needs a full diffcalc re-run after fixing.

Reply:
Oh dear.. luckily no sheets were generated for the latest changes. osu-tools isn't using these attribute IDs and the huismetbenen website that is used for users also doesn't use these so I'm not concerned about the values - but obviously, this needs to be fixed before anything which *does* use these can be relied upon.